### ISSUE TITLE
Uncaught TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by ghost** [callahanrts/slack-chat#19]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.211.0
**System**: Ubuntu 15.04
**Thrown From**: [slack-chat](https://github.com/callahanrts/slack-chat) package, v3.2.0
### Stack Trace

Uncaught TypeError: Cannot read property 'replace' of undefined

```
At /home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:176

TypeError: Cannot read property 'replace' of undefined
  at ChatLogView.module.exports.ChatLogView.message (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:176:16)
  at ChatLogView.message (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:2:1)
  at ChatLogView.module.exports.ChatLogView.parseMessage (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:171:13)
  at ChatLogView.parseMessage (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:2:1)
  at ChatLogView.module.exports.ChatLogView.messageElement (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:68:9)
  at ChatLogView.messageElement (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:2:1)
  at ChatLogView.module.exports.ChatLogView.addMessage (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:30:27)
  at ChatLogView.addMessage (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:2:1)
  at ChatLogView.module.exports.ChatLogView.initialize (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:26:6)
  at ChatLogView.View (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at ChatLogView.ScrollView (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/scroll-view.js:12:47)
  at new ChatLogView (/home/peduxe/.atom/packages/slack-chat/lib/views/chat/chat-log-view.coffee:19:3)
  at /home/peduxe/.atom/packages/slack-chat/lib/views/chat-view.coffee:42:26
  at /home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/lib/app.js:146:18
  at done (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/needle/lib/needle.js:355:7)
  at PassThrough.<anonymous> (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/needle/lib/needle.js:523:9)
  at emitNone (events.js:72:20)
  at PassThrough.emit (events.js:163:7)
  at _stream_readable.js:891:16
  at process._tickCallback (node.js:357:13)

```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "script",
      "polymer-snippets",
      "tool-bar-main",
      "filetype-color",
      "pigments"
    ],
    "themes": [
      "one-dark-ui",
      "atom-material-syntax"
    ]
  },
}
```
### Installed Packages

``` coffee
# User
.bin, vundefined
Sublime-Style-Column-Selection, v1.3.0
atom-material-syntax, v0.1.1
atom-ternjs, v0.6.4
atom-typescript, v4.6.2
autoclose-html, v0.18.0
autocomplete-paths, v1.0.2
color-picker, v2.0.7
emmet, v2.3.10
file-icons, v1.5.8
highlight-selected, v0.10.1
linter, v1.0.1
linter-csslint, v0.0.13
linter-htmlhint, v0.0.16
linter-jshint, v0.1.6
linter-php, v0.0.15
slack-chat, v3.2.0
tabs-to-spaces, v0.11.0
terminal-panel, v1.14.1

# Dev
No dev packages
```

**Comment by ghost** [comment]
Second exception throw after authorizing the app.

**Comment by callahanrts** [comment]
Should be resolved in [v3.2.1](https://github.com/callahanrts/slack-chat/tree/v3.2.1).
